test/tests/parallel/set1/spam_virus/rspamd_full.bats

```diff
@@ -43,16 +43,24 @@ function setup_file() {
   _wait_for_service postfix
   _wait_for_smtp_port_in_container
 
-  # We will send 3 emails: the first one should pass just fine; the second one should
-  # be rejected due to spam; the third one should be rejected due to a virus.
-  export MAIL_ID1=$(_send_email_and_get_id --from 'rspamd-pass@example.test' --data 'rspamd/pass')
-  export MAIL_ID2=$(_send_email_and_get_id --from 'rspamd-spam@example.test' --data 'rspamd/spam')
-  export MAIL_ID3=$(_send_email_and_get_id --from 'rspamd-virus@example.test' --data 'rspamd/virus')
-  export MAIL_ID4=$(_send_email_and_get_id --from 'rspamd-spam-header@example.test' --data 'rspamd/spam-header')
-
-  for ID in MAIL_ID{1,2,3,4}; do
-    [[ -n ${!ID} ]] || { echo "${ID} is empty - aborting!" ; return 1 ; }
-  done
+  # ref: https://rspamd.com/doc/gtube_patterns.html
+  local GTUBE_SUFFIX='*C4JDBQADN1.NSBN3*2IDNEN*GTUBE-STANDARD-ANTI-UBE-TEST-EMAIL*C.34X'
+
+  # We will send 4 emails:
+  # 1. The first one should pass just fine
+  _send_email_and_get_id MAIL_ID_PASS
+  # 2. The second one should be rejected (GTUBE pattern)
+  _send_email_and_get_id MAIL_ID_REJECT --expect-rejection --body "XJS${GTUBE_SUFFIX}"
+  # 3. The third one should be rejected due to a virus (ClamAV EICAR pattern)
+  # shellcheck disable=SC2016
+  _send_email_and_get_id MAIL_ID_VIRUS --expect-rejection \
+    --body 'X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*'
+  # 4. The fourth one will receive an added header (GTUBE pattern)
+  _send_email_and_get_id MAIL_ID_HEADER --body "YJS${GTUBE_SUFFIX}"
```

Comment:
Optional, lifts the `--body` test string values to variables.

For GTUBE and EICAR, I had mentioned a convenience method as an alternative. While mostly unnecessary, the intent was for shared use with SpamAssassin / ClamAV related test-suite files to leverage too. They could likewise drop the `.txt` files in favor of using `--body` AFAIK?

```suggestion
  # ref: https://rspamd.com/doc/gtube_patterns.html
  local GTUBE_SUFFIX='*C4JDBQADN1.NSBN3*2IDNEN*GTUBE-STANDARD-ANTI-UBE-TEST-EMAIL*C.34X'

  # Well-known test strings:
  local EICAR='X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*'
  local GTUBE="XJS${GTUBE_SUFFIX}"
  # Rspamd specific variation:
  local GTUBE_ADD_HEADER="YJS${GTUBE_SUFFIX}"

  # We will send 4 emails:
  # 1. The first one should pass just fine
  _send_email_and_get_id MAIL_ID_PASS
  # 2. The second one should be rejected (GTUBE pattern)
  _send_email_and_get_id MAIL_ID_REJECT --expect-rejection --body "${GTUBE}"
  # 3. The third one should be rejected due to a virus (ClamAV EICAR pattern)
  _send_email_and_get_id MAIL_ID_VIRUS --expect-rejection --body "${EICAR}"
  # 4. The fourth one will receive an added header (GTUBE pattern)
  _send_email_and_get_id MAIL_ID_HEADER --body "${GTUBE_ADD_HEADER}"
```

Reply:
I'm not sure if I like that. I see no benefit here really...